src/ape/utils.py

```diff
@@ -302,8 +305,19 @@ def get_all_files_in_directory(path: Path) -> List[Path]:
     return [path]
 
 
+class AbstractDataClassMeta(DataClassMeta, ABCMeta):
```

Comment:
I may have missed this, but what exactly is this for?

Reply:
This was moved over from `ape.api.base`, and is used for the following line 312:
```py
abstractdataclass = partial(dataclass, kwargs=True, meta=AbstractDataClassMeta)
```